### PR TITLE
libgsm 1.0.16

### PR DESCRIPTION
**Pull request by s172262**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/libgsm.rb [hunk 1]
```diff
@@ -31,8 +31,10 @@ def install
     man1.mkpath
     man3.mkpath
 
+    inreplace "Makefile", "libgsm.1.0.13.dylib", "libgsm.1.0.16.dylib"
```
Comment (ilovezfs) [inline Formula/libgsm.rb]: We'll need this reported upstream and a new release tarball will likely be needed.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/libgsm.rb (hunk 1 above)
Comment (s172262) [inline Formula/libgsm.rb]: Original Makefile by upstream don't have string of "libgsm.1.0.13.dylib".
It was written by the Patch. And Upstream said that They won't provide _dylib_.

Comment (ilovezfs) [inline Formula/libgsm.rb]: >And Upstream said that they won't provide dylib.

Do you have a reference for this? That may be sufficient reason to remove this from Homebrew.

>Original Makefile by upstream don't have string of "libgsm.1.0.13.dylib".
It was written by the Patch.

The patch should be updated accordingly and submitted to https://github.com/Homebrew/formula-patches.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
@s172262 any update here?

**Comment by ilovezfs** [comment]
@s172262 thanks for this PR. Please feel free to reopen it when the review feedback has been addressed.

**Pull request closed without merging**